### DISCUSSION TITLE
Disable Livewire 3 popup page

### DISCUSSION THREAD

**Discussion by cwkhawand** [livewire/livewire#6857] [1. Help] [answered]
Hi there,

I'd like to know if there's a similar feature as window.livewire.onError but for Livewire 3?
For example, when authorizing a user to a specific action when submitting a form, I'd rather show a text error in the form instead of showing the livewire popup window with 403 | Forbidden.

**Comment by joshhanley** [comment, marked as answer]
@cwkhawand have a look at this https://livewire.laravel.com/docs/javascript#request-hooks

It should allow you to customise how the error response is handled.

Hope this helps!

**Reply by cwkhawand** [reply]
Thank you!
